charts/cli-test.just

```diff
@@ -0,0 +1,193 @@
+_default:
+  @just --list
+
+cli-test *ARGS:
+  @just {{ARGS}}
+
+deploy:
+  @just deploy cluster
+  @just deploy ingress-controller
+  @just wait-for-ingress-controller
+  @just deploy astria-local
+
+command tag *ARGS:
+  docker run --rm --network host "ghcr.io/astriaorg/astria-cli{{ if tag != '' { replace(':#', '#', tag) } else { '' } }}" {{ ARGS }}
+
+fundedAccount := "astria1rsxyjrcm255ds9euthjx6yc3vrjt9sxrm9cfgm"
+privKey1 := "2bd806c97f0e00af1a1fc3328fa763a9269723c8db8fac4f93af71db186d6e90"
+privKey2 := "dfa7108e38ab71f89f356c72afc38600d5758f11a8c337164713e4471411d2e0"
+ibcAddAddress := "astria13ahqz4pjqfmynk9ylrqv4fwe4957x2p0h5782u"
+pubKey := "88787e29db8d5247c6adfac9909b56e6b2705c3120b2e3885e8ec8aa416a10f1"
+newPower := "100"
+originalPower := "10"
+sequencerRpc := "http://rpc.sequencer.localdev.me"
+sequencerChainId := "sequencer-test-chain-0"
+asset := "test"
+defaultTag := "latest"
+
+validate-output output expected_format:
+  #!/usr/bin/env bash
+  set -e 
+  
+  if echo "$output" | perl -0777 -ne 'exit 0 if /'"$expected_format"'/; exit 1'; then
+    printf "\e[32m✔\e[0m Output matches the expected format.\n"
+  else
+    printf "❌ Output does not match the expected format.\n"
+    exit 1
+  fi
+
+# Test the `create-account` command
+create-account tag=defaultTag:
+  #!/usr/bin/env bash
+  set -e 
+
+  printf "\nRunning the 'sequencer account create' command...\n"
+  output=$(just cli-test command {{ tag }} sequencer account create)
+  expected_format="^Create Sequencer Account\n\nPrivate Key:\s+\"[0-9a-f]{64}\"\nPublic Key:\s+\"[0-9a-f]{64}\"\nAddress:\s+\"[0-9a-f]{40}\"$"
+  echo "Output:\n$output"
+  just cli-test validate-output "$output" "$expected_format"
+
+# Test the `bech32m` command
+bech32m tag=defaultTag:
+  #!/usr/bin/env bash
+  set -e 
+
+  # make a new account to get the address bytes
+  output=$(just cli-test command {{ tag }} sequencer account create)
```

Comment:
Maybe should be a default address in bytes configured instead of creating an account? create account might change to print a bech32m prefixed address instead of bytes